<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetModel1</class>
 <widget class="QWidget" name="WidgetModel1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
   <item row="1" column="1">
    <widget class="XyProcessLog" name="tableWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="VideoWidget" name="widgetVideo" native="true">
     <property name="maximumSize">
      <size>
       <width>640</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="XyResultLabel" name="labelTips">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>点击检测设备环境是否正常</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XyResultLabel</class>
   <extends>QLabel</extends>
   <header>XyResultLabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XyProcessLog</class>
   <extends>QWidget</extends>
   <header location="global">XyProcessLog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
